File: zeyigou_sellergoods_interface/src/main/java/com/zeyigou/sellergoods/service/BrandService.java
package com.zeyigou.sellergoods.service;

import com.zeyigou.pojo.PageResult;
import com.zeyigou.pojo.Result;
import com.zeyigou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAll();


    PageResult search(int page, int size, TbBrand brand);

    Result add(TbBrand brand);

    Result update(TbBrand brand);

    Result delete(Long[] selectIds);

    List<Map> findBrandList();
}

File: zeyigou_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http){
        //查询所有品牌列表
        this.findAll=()=>{
                return $http.get("../brand/list.do")
        }
        //条件分页查询
        this.search=(page,size,searchEntity)=>{
                return $http.post("../brand/search.do?page="+page+"&size="+size,searchEntity)
        }
        //修改
        this.update=(entity)=>{
             return  $http.post("../brand/update.do",entity)
        }
        //添加
        this.add=(entity)=>{
                return $http.post("../brand/add.do",entity)
        }
        //删除
        this.del=(selectIds)=>{
               return  $http.post("../brand/delete.do?ids="+selectIds)
        }
        //查询品牌
        this.findBrandList=()=>{
                return $http.post("../brand/findBrandList.do")
        }
})